cou/utils/juju_utils.py

```diff
@@ -136,22 +137,10 @@ class COUModel:
 
     def __init__(self, name: Optional[str]):
         """COU Model initialization with name and juju.model.Model."""
-        self._model = Model(max_frame_size=JUJU_MAX_FRAME_SIZE)
+        self._juju_data = FileJujuData()
+        self._model = Model(max_frame_size=JUJU_MAX_FRAME_SIZE, jujudata=self._juju_data)
```

Comment:
Can't we use the property directly?
```suggestion
        self._model = Model(max_frame_size=JUJU_MAX_FRAME_SIZE, jujudata=self.juju_data)
```